src/__test__/homePageBottom.test.js

```diff
@@ -0,0 +1,125 @@
+import { render, cleanup } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import * as pokemonAction from '../redux/pokemon/actionsCreator';
+import store from '../redux/configStore';
+import ShowPokemon from '../components/homePageBottom';
+import DetailInfo from '../components/details';
+import Navbar from '../components/navbar';
+
+afterEach(() => {
+  cleanup();
+});
+
+describe('Render components to the DOM', () => {
+  test('Render Navbar', () => {
+    render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <Navbar />
+        </Provider>
+      </BrowserRouter>,
+    );
+  });
+
+  test('Render ShowPokemon component', () => {
+    const nameToPath = 'pl1-1';
+    const pokemon = { name: 'Ampharos' };
+    render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <ShowPokemon nameToPath={nameToPath} pokemon={pokemon.name} />
+        </Provider>
+      </BrowserRouter>,
+    );
+  });
+
+  test('Render Detail Component', () => {
+    const data = {//eslint-disable-line
+      name: 'Ampharos',
+      id: 'pl1-1',
+      pokedex: 181,
+      hp: '180',
+      type: 'Lightning',
+      attack: [{
+        name: 'Gigavolt',
+      }],
+      rarity: 'Rare Holo',
+      image: 'https://images.pokemontcg.io/pl1/1.png',
+      price: {
+        averageSellPrice: 2.69,
+      },
+    };
+    render(
+      <BrowserRouter>
+        <Provider store={store}>
+          <DetailInfo data={
+            {
+              name: 'Ampharos',
+              id: 'pl1-1',
+              pokedex: 181,
+              hp: '180',
+              type: 'Lightning',
+              attack: [{
+                name: 'Gigavolt',
+              }],
+              rarity: 'Rare Holo',
+              image: 'https://images.pokemontcg.io/pl1/1.png',
+              price: {
+                averageSellPrice: 2.69,
+              },
+            }
+          }
+          />
```

Comment:
[OPTIONAL] I think there is a bug somewhere with this test, as the console it's logging a PropType warning. It's possibly related to the default prop or the props being passed to the DetailInfo component.

You can check a screenshot of the warning here:
![image](https://user-images.githubusercontent.com/79114233/147284020-7b78eb28-d3ef-4557-ba23-cbe4ff5f999d.png)

Reply:
That's true, I'll work on it! thanks Daniel 🤓